### DISCUSSION TITLE
Getting Back More Classic Applets in Control Panel

### DISCUSSION THREAD

**Discussion by vippium** [valinet/ExplorerPatcher#1225] [Ideas]
1. There should be a option to return back the classic personalization in Control Panel with full support. We can add it by Winaero but it doesn't give the full support and redirects to Settings in last. StartAllBack provides the full support of it. It should also be in EP too, as this is very useful for some classic lover users. 

2. Getting back the "View Installed updates" option in Control Panel. SAB is already stepped for it but EP still not. I think that there is a universal option to get them back instead of Single-single for each item. I don't think that anyone wants to use the Settings app by leaving our precious Control Panel.

Dev's can add this as option like they did with About Page, Program & Features, etc.

**Comment by Ben2557** [comment]
Winaero Tweaker :
![image](https://user-images.githubusercontent.com/91072875/231688046-2e2ba172-bc32-4d89-bbc8-a3ece9994681.png)

More details : https://winaero.com/personalization-control-panel-windows-10/?utm_source=software&utm_medium=in-app&utm_campaign=winaerotweaker&utm_content=controlpanelpersonalization

**Reply by vippium** [reply]
I know this...but in this, u have different options in CP which is not good. And if u use 1 and select any option in that, it will redirect to settings.
What if we have a full implementation of it. You can check the SAB, which restores it completely.

**Reply by Ben2557** [reply]
Can you show me a screenshot/link on how StartAllBack works ?

**Reply by vippium** [reply]
I will share a full clip of both (winaero and SAB working) apps soon.

**Comment by vippium** [comment]
Working of Winaero Personalization :- 

https://user-images.githubusercontent.com/93650504/231705760-bc57c36e-c226-4294-bf3b-f1061efae7a4.mp4

Working of SAB Personalization :- 

https://user-images.githubusercontent.com/93650504/231706035-544a5ff4-e25d-4d58-a5da-df4e2d127ffe.mp4
